### ISSUE TITLE
Auto indentation doesn't work

### ISSUE THREAD

**Issue by zyavrik** [Anviking/Chromatism#14]
Steps to reproduce:
1. Type:
function main {
__line1;
2. Press Enter.

Auto indentation won't work.

**Comment by mbcoder17** [comment]
I am still having this issue...does this feature work with Swift?

**Comment by Anviking** [comment]
@mbcoder17 Hmm, actually I think auto-indentation might have been lost with the swift rewrite in v0.3

**Comment by mbcoder17** [comment]
Can it be brought back? :)

**Comment by Anviking** [comment]
Yeah, I think was something really really simple. With school (basically) finished and an upcoming WWDC I'll probably start looking at things again soon.

**Comment by mbcoder17** [comment]
Great, thanks! I'll see if I can figure it out myself too. Maybe I'll see you at WWDC! ;)

**Comment by mbcoder17** [comment]
Have you had time to implement this in the swift version? Great library!

**Comment by cyrilbarthelet** [comment]
The current version is in Swift

**Comment by mbcoder17** [comment]
@cyrilbarthelet Thank you, I know that. Please read the entire discussion next time before responding. I was asking if the auto-indentation was implemented in the swift version.

**Comment by Anviking** [comment]
@mbcoder17 Have stopped working on Chromatism, might rewrite it some time in the distant future. If you like, I think [this](https://github.com/Anviking/Chromatism/blob/cbb3b89c25cd49f9f6d05e69e067dde168346de1/Chromatism/Chromatism/JLTokenizer.m#L99L125) was the latest implementation of it :)

**Comment by mbcoder17** [comment]
@Anviking I am sad to hear that, I think it is a great project and it has helped me a lot with one of my own projects! :) I was recently looking through previous versions and was able to get the indentation working. I have also made other additions such as line numbers, maybe I will fork my own version if you don't mind! Good luck on other projects!
